tools/tools_common.h

```diff
@@ -115,11 +115,11 @@ disableLogging(dht::DhtRunner& dht)
 /**
  * Converts std::chrono::duration to floating-point seconds.
  */
-template <class DT>
+/*template <class DT>
```

Comment:
This has certainly been missed. The function has been moved to `include/opendht/utils.h`. Thanks for noticing.